### PR TITLE
fix(ngModelOptions): allow sharing options between multiple inputs

### PR DESCRIPTION
**Pull request by shahata**
Closes #10667

### CODE REVIEW

**Comment by lgalfaso** [comment]
landed as 9c9c6b3fe4edfe78ae275c413ee3eefb81f1ebf6

File: src/ng/directive/ngModel.js [hunk 1]
```diff
@@ -1215,7 +1215,7 @@ var ngModelOptionsDirective = function() {
     restrict: 'A',
     controller: ['$scope', '$attrs', function($scope, $attrs) {
       var that = this;
-      this.$options = $scope.$eval($attrs.ngModelOptions);
+      this.$options = angular.copy($scope.$eval($attrs.ngModelOptions));
```
Comment (jbedard) [inline src/ng/directive/ngModel.js]: Really minor, and a bit late, but this should just be `copy`...

Comment (lgalfaso) [inline src/ng/directive/ngModel.js]: You are right, should be able to make the change in about an hour (unless
someone beats me)
>
>

Comment (lgalfaso) [inline src/ng/directive/ngModel.js]: done and thanks

Comment (shahata) [inline src/ng/directive/ngModel.js]: Thanks & Sorry. I always keep forgetting not to use `angular.*` :(

**Pull request closed without merging**